BUILD.bazel

```diff
@@ -0,0 +1,164 @@
+# Allow any ts_library rules in this workspace to reference the config
+exports_files(["tsconfig.json"], visibility = ["//:__subpackages__"])
+
+package(default_visibility = ["//visibility:public"])
+
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary", "nodejs_test")
+load("@npm_bazel_typescript//:index.bzl", "ts_project", "ts_config")
+load("@npm_bazel_labs//:index.bzl", "protobufjs_ts_library")
+
+proto_library(
+    name = "google_protobuf_protos",
```

Comment:
The idiomatic way for bazel is to have BUIDL.bazel file in each sub-folder (which turns that folder and all its subfolders without their own BUILD.bazel files into a "package" from bazel's perspective).
For example, please consider putting proto_library targets which compile stuff under  `node_modules/google-gax/build/protos` into a BUIDL.bazel file under same directory (i.e. `node_modules/google-gax/build/protos`).

Reply:
Let me postpone this as a possible refactor - I see your point but for now I would prefer not to introduce any `BUILD.bazel` files to `google-gax` (the reason for that is that `build/protos` folder in `google-gax` is auto-generated and I cannot just put stuff there; also, it's a user-facing folder).